src/model/AreaDataSource.ts

```diff
@@ -108,6 +108,12 @@ export default class AreaDataSource extends MongoDataSource<AreaType> {
           $set: {
             'climbs.gradeContext': '$gradeContext' // manually set area's grade context to climb
           }
+        },
+        {
+          $set: {
+            climbs: { $sortArray: { input: '$climbs', sortBy: { 'metadata.left_right_index': 1 } } },
```

Comment:
`$sortArray` requires Mongo 5.2 and up. Otherwise we need something like this: https://stackoverflow.com/questions/15388127/mongodb-sort-inner-array